doc/topics/windows/multi-minion.rst

```diff
@@ -0,0 +1,164 @@
+
+The steps required to do this are as follows:
+
+1. Create new root_dir
+2. Set root_dir permissions
+3. Create directory structure
+4. Write minion config
+
+We will now go through each of these steps in detail.
+
+.. note::
+
+    We have created a powershell script that will configure an additional minion
+    on the system for you. It can be found in the root of the Salt installation.
+    The script is named ``multi-minion.ps1``. You can get help on how to use the
+    script by running the following in a PowerShell prompt:
+
+    ``Get-Help .\multi-minion.ps1 -Detailed``
+
+1. Create new ``root_dir``
+--------------------------
+
+The minion requires a root directory to store config, cache, logs, etc. The user
+must have full permissions to this directory. The easiest way to do this is to
+put the ``root_dir`` in the Local AppData directory (``$env:LocalAppData``).
+
+.. code-block:: powershell
+
+    New-Item -Path "$env:LocalAppData\Salt Project\Salt" -Type Directory
+
+2. Set ``root_dir`` permissions
+-------------------------------
+
+The user running Salt requires full access to the ``root_dir``. If you have
+placed the root_dir in a location that the user does not have access to, you'll
+need to give the user full permissions to that directory.
+
+.. code-block:: powershell
+
+    $RootDir = "<new root_dir location>"
+    $User    = "<user running salt>"
+    $acl = Get-Acl -Path "$RootDir"
+    $access_rule = New-Object System.Security.AccessControl.FileSystemAccessRule($User, "Modify", "Allow")
+    $acl.AddAccessRule($access_rule)
+    Set-Acl -Path "$RootDir" -AclObject $acl
+
+3. Create directory structure
+-----------------------------
+
+Salt expects a certain directory structure to be present to avoid unnecessary
+messages in the logs. This is usually handled by the installer. Since we're
+running our own instance, we need to do it. Make sure the following directories
+are present:
+
+  - root_dir\\conf\\minion.d
+  - root_dir\\conf\\pki
+  - root_dir\\var\\log\\salt
+  - root_dir\\var\\run
+  - root_dir\\var\\cache\\salt\\minion\\extmods\\grains
+  - root_dir\\var\\cache\\salt\\minion\\proc
+
+.. code-block:: powershell
+
+    $RootDir = "<new root_dir location>"
+    $cache_dir = "$RootDir\var\cache\salt\minion"
+    New-Item -Path "$RootDir\conf" -Type Directory
+    New-Item -Path "$RootDir\conf\minion.d" -Type Directory
+    New-Item -Path "$RootDir\conf\pki" -Type Directory
+    New-Item -Path "$RootDir\var\log\salt" -Type Directory
+    New-Item -Path "$RootDir\var\run" -Type Directory
+    New-Item -Path "$cache_dir\extmods\grains" -Type Directory
+    New-Item -Path "$cache_dir\proc" -Type Directory
+
+4. Write minion config
+----------------------
+
+The minion will need is own config, separate from the system minion config. This
+config tells the minion where everything is as well as defines the master and
+minion id. Create a minion config file named ``minion`` in the conf directory.
```

Comment:
**Suggestion (grammar, style, and prasing, blocking):** Small grammar error "is own." Also, this is a minor thing, but our style guide says to use the present tense: https://developers.google.com/style/tense

I also recommend some minor rephrasing.

Suggested fix:
```
The minion needs its own config, separate from the system minion config. This
config tells the minion where everything is located in the file structure and also
defines the master and minion id.
```